packages/orchestration/test/service.test.ts

```diff
@@ -207,3 +210,178 @@ test('makeAccount accepts opts (version, ordering, encoding)', async t => {
     );
   }
 });
+
+test('.close() sends a downcall to the ibc bridge handler', async t => {
```

Comment:
tangential, but shouldn't this test suite be `exos/cosmos-interchain-service.test.ts` ?